cmd/argo/commands/lint.go

```diff
@@ -37,32 +38,41 @@ func NewLintCommand() *cobra.Command {
 						return err
 					}
 				}
-				fmt.Printf("%s is valid\n", file)
+				fmt.Printf("%s is valid \u2713\n ", file)
 				return nil
 			}
-
 			for _, file := range args {
 				stat, err := os.Stat(file)
 				errors.CheckError(err)
 				if stat.IsDir() {
-					err := filepath.Walk(file, func(path string, info os.FileInfo, err error) error {
+					filepath.Walk(file, func(path string, info os.FileInfo, err error) error {
 						fileExt := filepath.Ext(info.Name())
 						switch fileExt {
 						case ".yaml", ".yml", ".json":
 						default:
 							return nil
 						}
-						return lint(path)
+						err = lint(path)
+						if err != nil {
+							unvalidWf = fmt.Errorf("unvalid workflow/workflows found")
+							log.Warn(fmt.Errorf("%w \u2717", err))
+						}
+						return nil
```

Comment:
We still want the error to be returned from the `filepath.Walk` function. It's ok to show the warning and return the original error.

Reply:
I had issues returning the error resulted in that the ```filepath.Walk ``` broke execution and did not complete for folders. I will verify this again.

Reply:
[Here](https://golang.org/pkg/path/filepath/#WalkFunc) is the documentation on file path.Walk as I understand from ``` ...  In the case of an error, the info argument will be nil. If an error is returned, processing stops. The sole exception is when the function returns the special value SkipDir. ...``` I can not return an error if I like to continue the execution. Therefore I create a new error outside of the function which I use to capture that the file path.walk has not been successful for all files. Maybe I am missing something? @simster7.

Reply:
[Here](https://golang.org/pkg/path/filepath/#WalkFunc) is the documentation on filepath.WalkFunc. Which is used by 
```
func Walk(root string, walkFn WalkFunc) error
```
as I understand from ``` ...  In the case of an error, the info argument will be nil. If an error is returned, processing stops. The sole exception is when the function returns the special value SkipDir. ...``` I can not return an error if I like to continue the execution. Therefore I create a new error outside of the function which I use to capture that the file path.walk has not been successful for all files. Maybe I am missing something? @simster7.

Reply:
Alright, let's leave this as is then.